apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -161,6 +162,14 @@ defmodule Aecore.Chain.Worker do
     Logger.info(fn ->
       "Added block ##{new_block.header.height} with hash #{Base.encode16(new_block_hash)}, total tokens: #{inspect(total_tokens)}"
     end)
+
+    # Broadcasting notifications for new mined transaction(per account)
+    Enum.each(new_block.txs, fn(tx) -> 
```

Comment:
can we move this in the `broadcast_new_block_added_to_chain` function

Reply:
Do you mean to move in broadcast_new_mined_transaction?

Reply:
no, later in the chain worker you broadcast the block, when broadcasting the block in `broadcast_new_block_added_to_chain` you can broadcast the transactions as well

Reply:
we want to minimize the logic needed in the core